trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/hive/SortingColumn.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastrato.gravitino.trino.connector.catalog.hive;
+
+import static com.datastrato.gravitino.trino.connector.catalog.hive.SortingColumn.Order.ASCENDING;
+import static com.datastrato.gravitino.trino.connector.catalog.hive.SortingColumn.Order.DESCENDING;
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static io.trino.spi.connector.SortOrder.ASC_NULLS_FIRST;
+import static io.trino.spi.connector.SortOrder.DESC_NULLS_LAST;
+import static java.util.Locale.ENGLISH;
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.errorprone.annotations.Immutable;
+import io.trino.spi.connector.SortOrder;
+import java.util.Objects;
+
+@Immutable
+public class SortingColumn {
```

Comment:
If this is copied from somewhere, you should add reference here and update the LICENSE file.